dev/registry.sh

```diff
@@ -139,10 +141,12 @@ elif [ "$cmd" = "update" ]; then
     build_and_push $ROOT/images/istio-galley istio-galley latest
   fi
 
+  build_and_push $ROOT/images/downloader downloader latest
```

Comment:
nit: if there was no particular reason to move downloader up here, do you mind moving it back down (just to clean up the diff)